pkg/skaffold/build/cluster/pod.go

```diff
@@ -90,6 +90,13 @@ func (b *Builder) kanikoPodSpec(artifact *latest.KanikoArtifact, tag string, pla
 		addSecretVolume(pod, kaniko.DefaultSecretName, b.ClusterDetails.PullSecretMountPath, b.ClusterDetails.PullSecretName)
 	}
 
+	// Add secret for pulling kaniko images from a private registry
```

Comment:
Please add a test for this change in pod_test.go

Reply:
is that good?